language/Greek/strings.po

```diff
@@ -406,7 +406,7 @@ msgstr "Παράλειψη μενού επιλογής BIOS"
 
 #: src/frontend/mame/ui/submenu.cpp:44
 msgid "Skip software part selection menu"
-msgstr ""
+msgstr "Παράλειψη μενού επιλογής τμήματος λογισμικού"
```

Comment:
I’d like clarification on this message.

When you select a system that supports software lists, there are potentially two menus that can be shown.  In English, they are called the “software selection menu” where you select a software item from the software list, and the “software part selection menu” where you select which “part” of the software item to mount for a software item with multiple parts (e.g. if a software item has multiple disks, each disk is typically represented as a separate part).

Does this translation make it clear that the setting only controls whether the software part selection menu is skipped, not the software selection menu?

For example, if you select a Zorba system from the system selection menu:
![image](https://user-images.githubusercontent.com/9678431/146404818-63c50687-8ac1-4cf2-9956-b65278ba510c.png)

You first see the software selection menu, where I select a CP/M OS:
![image](https://user-images.githubusercontent.com/9678431/146404895-37d51139-4767-4777-bc9a-002ada40edd6.png)

Which leads to the software part selection menu (yes, this software list doesn’t have very good descriptions for the parts of the items):
![image](https://user-images.githubusercontent.com/9678431/146405068-99c98aa8-922a-4e23-b341-71f1d37fac57.png)

The setting that this message corresponds to only controls skipping the menu shown in the third screenshot here.

Reply:
Yes, the translation of the specified string refers to "τμήματος " ->(Τμήμα) which translates to "part" from english.  The "software selection menu" should translates to "μενού επιλογής λογισμικού" so it seems to be correct.

Reply:
Thanks for clarifying.